modules/token_merging/README.md

```diff
@@ -0,0 +1,61 @@
+# Token Merging for Stable Diffusion running with OpenVINO
+
+This is an OpenVINO adoped version of Token Merging method. The method is applied to PyTorch model before exporting to OpenVINO representation. It can be also stacked with 8-bit quantization to achieve a higher inference speed. 
```

Comment:
```suggestion
This is an OpenVINO adopted version of Token Merging method. The method is applied to PyTorch model before exporting to OpenVINO representation. It can be also stacked with 8-bit quantization to achieve a higher inference speed. 
```